lcls_tools/common/measurements/beam_charge.py

```diff
@@ -0,0 +1,11 @@
+from lcls_tools.common.devices.ict import ICT
+from lcls_tools.common.measurements.measurement import Measurement
+
+
+class BeamChargeMeasurement(Measurement):
+    name = "beam_charge"
+    ict_monitor: ICT
+
+    def measure(self) -> dict:
+        return {"bunch_charge_nC": self.ict_monitor.get_charge()}
+
```

Comment:
Assign this to a variable, as we'll prob add more things to the dict that gets returned?

Reply:
sure thats fine, but should be done in a device specific PR. This is just a simple placeholder

Reply:
If it's something we know we will do, I lean towards just adding it in now for style/format guidance to future contributors. At some point it will be hard to keep track of things.